src/ducks/categorization/services.spec.js

```diff
@@ -1,13 +1,26 @@
 import {
   getUniqueCategories,
   getAlphaParameter,
-  createLocalClassifier
+  createLocalClassifier,
+  localModel
 } from './services'
-import bayes from 'classificator'
 import { tokenizer } from '.'
-
-const mockLearn = jest.fn()
-jest.mock('classificator', () => jest.fn(() => ({ learn: mockLearn })))
+import { Transaction } from '../../models'
+
+jest
+  .spyOn(Transaction, 'queryAll')
+  .mockResolvedValue([
+    { label: 'AAAA BBBB', manualCategoryId: '200110' },
```

Comment:
It would be preferable to add an `amount` field in the learning transactions. Under the hood, these operations will pass through `getLabelWithTags()` that waits for an `amount`field. Currently the tests are certainly OK because the fallback of `getLabelWithTags()` if no amount is givent corresponds to a salary *and* because these operations actually correspond to a salary. But the tests will probably fail if we add some operations not supposed to be associated with `{ amount: 3001.71, label: 'AAAA BBBB' }`